### PR TITLE
Feature/load articles using js

### PR DESCRIPTION
**Pull request by vinayak-4**
Use javascript to load article and add load more article button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <script src="./script.js" defer></script>
```
Comment (Delwalt) [inline index.html:13]: what is defer ?

File: index.html [hunk 2]
```diff
@@ -19,10 +20,14 @@
...
-        <label for="nav-check" id="hamburger-button"><i class="fa fa-bars"></i></label>
-        <input id="nav-check" name="nav-check" type="checkbox">
+        <label for="nav-check" id="hamburger-button"
+          ><i class="fa fa-bars"></i
+        ></label>
+        <input id="nav-check" name="nav-check" type="checkbox" />
         <ul class="nav-item-container">
-          <label for="nav-check" id="mobile-close-btn"><i class="fa fa-times" aria-hidden="true"></i></label>
+          <label for="nav-check" id="mobile-close-btn"
+            ><i class="fa fa-times" aria-hidden="true"></i
```
Comment (Delwalt) [inline index.html:29]: what is `aria-hidden`?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+const articleContainer = document.querySelector(".article-container");
```
Comment (Delwalt) [inline script.js]: to select an element from dom always use ids

File: style.css [hunk 4]
```diff
@@ -233,7 +237,7 @@ img {
   .banner.container {
     column-gap: 15px;
   }
-  .read-more-btn {
+  .primary-btn {
```
Comment (Delwalt) [inline style.css:240]: why duplicate `primary-btn` class ?

File: index.html (hunk 1 above)
Comment (vinayak-4) [reply index.html:13]: Defer let the javascript file load in background and execute the file when the HTML file is done parsing

File: index.html (hunk 2 above)
Comment (vinayak-4) [reply index.html:29]: aria-hidden removes the element from accessibility tree means it tells the screen reader to ignore it

File: script.js (hunk 3 above)
Comment (vinayak-4) [reply script.js]: Okay i'll improve it

File: style.css (hunk 4 above)
Comment (vinayak-4) [reply style.css:240]: At first I needed only single button but now I added some more buttons so to style all buttons I renamed the class to primary-btn

Comment (vinayak-4) [reply style.css:240]: I checked the code again and the duplicate classes are separated by media queries

File: script.js (hunk 3 above)
Comment (vinayak-4) [reply script.js]: I have corrected it.

**Review by Delwalt — APPROVED** [review]

**Pull request merged**